src/ttkbootstrap/scrolled.py

```diff
@@ -272,7 +272,7 @@ def __init__(
 
         # content frame
         super().__init__(
-            master=self.container,
```

Comment:
This is required to make the frame scrollable.

However, in the `destroy()` method, if you can remove all method calls except for `super().destroy()`, then I think this fixes it.

Reply:
Thank you for the review, I will change and check the behavior.

Reply:
Fixed and checked with the given #656 and #673.